src/components/Card/Card.spec.js

```diff
@@ -2,17 +2,15 @@ import {render, screen} from '@testing-library/react';
 import Card from './Card';
 
 describe('Card', () => {
-  it('renders a card with a heading, one paragraph and two list items', () => {
+  it('renders a card with a heading and two paragraphs', () => {
     render(<Card />);
 
     const heading = screen.getByRole('heading');
     const description = screen.getByTestId('description');
-    const repeat = screen.getByTestId('repeat');
```

Comment:
In general, I would go with the [React Testing Library](https://testing-library.com/docs/queries/about#priority) and rather use `getByText` instead of adding a test id; but I think it's fine for now.

Reply:
I tried to use getByText, but since the text in the elements I wanted to test is dynamic, unfortunately that didn't work. But I will look into it once again.

Reply:
You can give the content as prop to your card as the following:
```js
render(<Card description='example description'/>);

const description = screen.getByText('example description');

expect(description).toBeInTheDocument();
```